index.js

```diff
@@ -0,0 +1,14 @@
+const express = require("express");
+const app = express()
+const router = require("./routes/route")
+
+app.set('view engine', 'ejs')
+const bodyParser = require("body-parser")
+app.use(bodyParser.urlencoded({ extended : true }))
```

Comment:
nice
biar lebih enak, sebenernya body parser itu ada di express-nya juga, jadi bisa langsung pake dari express dan hasilnya nggak berubah:
```suggestion
app.use(express.urlencoded({ extended : true }))
```